<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>378</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>213</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>213</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>213</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>213</green>
        <blue>213</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="cuadro_grafico">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>298</width>
      <height>199</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Tipo">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Seleccione la grafica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tipo_grafica">
       <item>
        <property name="text">
         <string>--</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EEG continuo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EEG una epoca</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ruta">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ruta de la se√±al: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ruta_senal"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="boton_graficar">
         <property name="text">
          <string>Graficar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="atras">
         <property name="text">
          <string>Atras</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="salir">
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
